### DISCUSSION TITLE
ExtEditorR issue with `snap`

### DISCUSSION THREAD

**Discussion by fschweitzer-ETHZ** [Frederick888/external-editor-revived#155] [Q&A]
# Description
Error message: 
"ExtEditorR encountered error from external editor
sh 1: /usr/bin/emacsclient: not found"

# Environment

- Ubuntu 24.04, GNU emacs 29.3. 

The error comes after the new TB was installed from snap. 

I already moved external_editor_revived.json to the new .mozilla/native-messaging-hosts/ dir under snap.
and reinstalled the extension. 
Version 1.1.0

**Comment by Frederick888** [comment]
You are welcome to share if you figure it out. Maybe it's similar to #103.

**Comment by fschweitzer-ETHZ** [comment]
I guess it's similar to [#103](https://github.com/Frederick888/external-editor-revived/discussions/103). But different from flatpak snap is much more restricted reg. spawn. I am not a programmer, so I can not figure it out. But I would be really interested to use extEditorR again.

**Comment by somescience** [comment]
This solved it for me: [How to Install Thunderbird as DEB (Not Snap) in Ubuntu 24.04
](https://ubuntuhandbook.org/index.php/2024/03/install-thunderbird-deb-ubuntu-2404/) I've had to use "For choice, you may add 3 more lines below".

**Comment by scfrank-pri** [comment]
I have figured out a [solution how extended_editor_revived 1.2.0. works with the Thunderbird snap](https://github.com/scfrank-pri/ext_editor_snap_config). Works for TB 128.7 in ubuntu 24.10
